lib/acl.c

```diff
@@ -227,6 +230,15 @@ struct rte_acl_field_def ipv4_defs[NUM_FIELDS_IPV4] = {
 		.input_index = PORTS_INPUT_IPV4,
 		.offset = sizeof(struct ipv4_hdr) + sizeof(uint16_t),
 	},
+	{
+		/* Enforce grouping into four bytes. */
+		.type = RTE_ACL_FIELD_TYPE_BITMASK,
+		.size = sizeof(uint32_t),
```

Comment:
Wouldn't it be easier to use ` sizeof(uint16_t)` since we are only testing for type and code?

Reply:
This is a [performance optimization](http://doc.dpdk.org/api/structrte__acl__field__def.html), which requires the input to be grouped into sets of 4 consecutive bytes.

Reply:
Thank you for the link.